src/js/elements/tutorial.js

```diff
@@ -0,0 +1,17 @@
+let wrapperClass = '.w-slider-mask';
+let slideClass = '.w-slide';
+
+export class Tutorial {
+    createSlides = (tutorialArr) => {
+        if (tutorialArr === null || tutorialArr.length <= 0) {
+            return;
+        }
+
+        tutorialArr.forEach((t, i) => {
```

Comment:
let's change `t` to `slide` which is more meaningful. `i` is a standard way of naming an iterator but the object should be named accordingly.

Reply:
done